### PR TITLE
Fixed javadoc inconsistencies + added .gitignore

### PR DESCRIPTION
**Pull request by slkasisto**
see https://github.com/actions-on-google/actions-on-google-java/issues/10 and https://github.com/actions-on-google/actions-on-google-java/issues/7

### CODE REVIEW

**Comment by slkasisto** [comment]
I signed the CLA.

File: src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt [hunk 1]
```diff
@@ -33,7 +33,7 @@ import com.google.api.services.dialogflow_fulfillment.v2.model.WebhookResponse
  * ActionsSdkApp):
  *
  * ``` Java
- * ResponseBuilder builder = getResponseBuilder()
+ * ResponseBuilder builder = getResponseBuilder(ActionRequest request);
```
Comment (taycaldwell) [inline src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt]: I assume you provided the type for clarification?

Correct example would be `ResponseBuilder builder = getResponseBuilder(request);`, assuming a ActionRequest variable named request.

File: src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt [hunk 2]
```diff
@@ -43,7 +43,7 @@ import com.google.api.services.dialogflow_fulfillment.v2.model.WebhookResponse
  * To end a conversation,
  *
  * ``` Java
- * ResponseBuilder builder = getResponseBuilder()
+ * ResponseBuilder builder = getResponseBuilder(ActionRequest request);
```
Comment (taycaldwell) [inline src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt]: I assume you provided the type for clarification?

Correct example would be `ResponseBuilder builder = getResponseBuilder(request);`, assuming a ActionRequest variable named request.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# General
```
Comment (taycaldwell) [inline .gitignore]: What are the contents of this .gitignore file based on?

Can we address this seaprately and make it it's own PR?

File: src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt (hunk 1 above)
Comment (slkasisto) [reply src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt]: good catch, thanks.  I updated

File: .gitignore (hunk 3 above)
Comment (slkasisto) [reply .gitignore]: Based on a sample MacOS .gitingore file I grabbed from somewhere on the internet.  Do you have a template that is preferred?

Comment (taycaldwell) [reply .gitignore]: @slkasisto The .gitignore contents of https://github.com/actions-on-google/dialogflow-facts-about-google-java/blob/master/.gitignore should suffice.

The one included in the PR has a lot of files that should never be present in the project directory.

Comment (slkasisto) [reply .gitignore]: @taycaldwell I replaced, thanks.

File: src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt [hunk 4]
```diff
@@ -33,7 +33,8 @@ import com.google.api.services.dialogflow_fulfillment.v2.model.WebhookResponse
  * ActionsSdkApp):
  *
  * ``` Java
- * ResponseBuilder builder = getResponseBuilder()
+ * ActionRequest request = ...;
+ * ResponseBuilder builder = getResponseBuilder(request);
```
Comment (taycaldwell) [inline src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt]: Can you apply these same changes to line 143?

Comment (slkasisto) [reply src/main/kotlin/com/google/actions/api/response/ResponseBuilder.kt]: Applied.

**Comment by taycaldwell** [comment]
Thank you for your contribution @slkasisto!
